### PR TITLE
Add initial tidy support

### PR DESCRIPTION
**Pull request by mrobinson**
Currently many directories are ignored, but we can add them as we tidy
things up

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/794)

### CODE REVIEW

File: servo-tidy.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    # There are many duplicated packages, which we probably need to review
+    # one by one.
+    "./Cargo.lock"
+]
+
+# Many directories are currently ignored while we tidy things up
+# gradually.
+directories = [
+   # Generated and upstream code combined with our own. Could use cleanup
+  "./target",
```
Comment (kvark) [inline servo-tidy.toml]: why do we need to tidy up the generated code?

Comment (mrobinson) [inline servo-tidy.toml]: The comment is indeed a little unclear, but this is a list of directories excluded from the tidy. We will gradually remove entries.

**Comment by glennw** [comment]
@mrobinson Needs a rebase. Also, the removal of replay/ dir has landed now - so we can remove that from the config file.

**Comment by mrobinson** [comment]
@glennw Thanks for the review.

**Pull request merged**